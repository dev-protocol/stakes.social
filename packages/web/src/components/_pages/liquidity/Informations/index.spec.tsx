import React from 'react'
import { render } from '@testing-library/react'
import { Informations } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/_pages/liquidity/geyser/hooks.ts')
jest.mock('src/fixtures/_pages/liquidity/uniswap-pool/hooks.ts')

describe(`${Informations.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Informations geyserAddress="0xdummy" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
