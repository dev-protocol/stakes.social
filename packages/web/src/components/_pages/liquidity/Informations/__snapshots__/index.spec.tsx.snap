// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Informations Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-jRQBWg jvcUAg"
    >
      Information
    </div>
    <div
      class="sc-bdvvtL birinC"
    >
      <div
        class="ant-card ant-card-bordered sc-gsDKAQ sc-dkPtRN dJtSgW iPDFlP"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Monthly Yield
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  25,920
                </span>
                
              </span>
              <span
                class="ant-statistic-content-suffix"
              >
                %
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered sc-gsDKAQ sc-hKwDye dJtSgW bjGuob"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Reward Multiplier
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  1
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .0
                </span>
              </span>
              <span
                class="ant-statistic-content-suffix"
              >
                X / 5X
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered sc-gsDKAQ sc-eCImPb dJtSgW ezcUUs"
      >
        <div
          class="ant-card-body"
        >
          <div
            class="ant-statistic"
          >
            <div
              class="ant-statistic-title"
            >
              Accrued Rewards
            </div>
            <div
              class="ant-statistic-content"
            >
              <span
                class="ant-statistic-content-value"
              >
                <span
                  class="ant-statistic-content-value-int"
                >
                  
                  0
                </span>
                <span
                  class="ant-statistic-content-value-decimal"
                >
                  .00
                </span>
              </span>
              <span
                class="ant-statistic-content-suffix"
              >
                DEV
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
